sites/lib/area.js

```diff
@@ -85,10 +79,7 @@ export default {
     general: {
       label: 'Content Widgets',
       widgets: {
-        '@apostrophecms/rich-text': {
-          toolbar: tiptapTools.all,
-          styles: tiptapStyles.all
-        },
+        '@apostrophecms/rich-text': {},
```

Comment:
@ValJed should it have been removed in https://github.com/apostrophecms/starter-kit-assembly-hospitality/pull/9?